.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do we need this file? if it's autogenerated, can we add it to `.gitignore`?

Reply:
I tend to just check Xcode-managed files in without worrying too much about what changed and why.